airflow/gcp/example_dags/example_bigquery.py

```diff
@@ -265,3 +265,7 @@
     execute_query_external_table >> bigquery_to_gcs >> delete_dataset
     create_table >> create_view >> delete_view >> delete_table >> delete_dataset
     create_dataset_with_location >> create_table_with_location >> delete_dataset_with_location
+
+if __name__ == '__main__':
+    dag.clear(reset_dag_runs=True)
+    dag.run()
```

Comment:
Why do we need this?

Reply:
With this, you can debug this DAG from IDE level (by debugging this file) using DebugExecutor. Highly recommended for operators development 🚀

Reply:
👍